plugins/org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/AdvancedNewProjectPage.xtend

```diff
@@ -282,6 +314,14 @@ class AdvancedNewProjectPage extends WizardPage {
 		createWebProject.selection
 	}
 
+	def boolean isCreateSdkProject() {
+		createSDKProject.selection
```

Comment:
Also here, return false if the whole section is disabled. The same for isCreateP2Project